src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.model.ts

```diff
@@ -58,6 +58,7 @@ export const buildModelledProperties = async ({
         // Get ADT Models
         const modelDict = (await adapter.getAllAdtModels()).getData()
             .parsedModels;
+        console.log('modelDict', modelDict);
```

Comment:
can we use `useLogger` instead for all console.logs around? @msnyder-msft this is the hook that we should use for any type of logging purpose from now on right? Not the utils method you created before this hook `getDebugLogger`?

Reply:
I didn't need these console.logs thought I had removed them, nice catch